ocamltest/ocaml_actions.ml

```diff
@@ -450,11 +459,51 @@ let ocamlopt_opt = Actions.make
   "ocamlopt.opt"
   (compile_test_program Builtin_variables.program2 ocamlopt_opt_compiler)
 
+let run_expect_once ocamlsrcdir input_file principal log env =
+  let expect_flags = try Sys.getenv "EXPECT_FLAGS" with Not_found -> "" in
+  let repo_root = "-repo-root " ^ ocamlsrcdir in
+  let principal_flag = if principal then "-principal" else "" in
+  let commandline =
+  [
+    expect_command ocamlsrcdir;
+    expect_flags;
+    flags env;
+    repo_root;
+    principal_flag;
+    input_file
```

Comment:
The original invocation command

```shell
TERM=dumb $(EXPECT_TEST) $(EXPECT_FLAGS) -repo-root $(OTOPDIR) $$file
```

also starts with a `TERM=dumb` setting to control the behavior of toplevel location printing (I suppose). Is it reproduced here, for example by being globally set by ocamltest, or by `Actions_helpers.run_cmd`?